### PR TITLE
Set default WebDriver path to `/`

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

Having `/wd/hub` as default path is a relict from the old days where people would use Selenium via JsonWireProtocol to automate browser. These days the default path from most of the driver is `/`.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

Breaking change to land in v6.

### CODE REVIEW

File: packages/webdriver/tests/request.test.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ describe('webdriver request', () => {
 
     describe('createOptions', () => {
         it('fails if command requires sessionId but none given', () => {
-            const req = new WebDriverRequest('POST', '/wd/hub/session/:sessionId/element')
```
Comment (mgrybyk) [inline packages/webdriver/tests/request.test.js]: Shall we move path to constant in the top of the file to reduce possible refactoring in later on?

File: packages/wdio-cli/src/templates/wdio.conf.tpl.ejs [hunk 2]
```diff
@@ -24,16 +24,6 @@ exports.config = {
     path: '<%= answers.path %>',<% } else if (answers.services.includes('appium')) { %>
     port: 4723,<% }
 
-    if(answers.services.includes('chromedriver')) { %>
```
Comment (mgrybyk) [inline packages/wdio-cli/src/templates/wdio.conf.tpl.ejs]: Do you think it makes sense now to set default path to wd/hub if selenium standalone is used? At least until v4 is out (no idea when is it going to happen)

File: packages/webdriverio/tests/__mocks__/request.js [hunk 3]
```diff
@@ -49,7 +49,7 @@ const requestMock = jest.fn().mockImplementation((params, cb) => {
     }
 
     switch (params.uri.path) {
-    case '/wd/hub/session':
```
Comment (mgrybyk) [inline packages/webdriverio/tests/__mocks__/request.js]: Having path in constant might help here as well

**Comment by christian-bromann** [comment]
@mgrybyk I added an `if` clause in `request.js` that checks for an HTML body that contains: `Whoops! The URL specified routes to this help page.` which is the case for someone using WebdriverIO v6 but with a Selenium Standalone v3.x. It will notify the user to change the path to `/wd/hub`.

I also set `/wd/hub` as default when user is running tests on cloud provider as they still seem to run on the old path.

**Comment by erwinheitzman** [comment]
Looking good to me. I can test out this branch with some services if you'd like to make sure we don't break anything.

**Pull request merged**